lib/dry_twitter/registration/persist.rb

```diff
@@ -1,12 +1,19 @@
 require "dry/transaction/operation"
+require "dry_twitter/import"
 
 module DryTwitter
   module Registration
     class Persist
       include Dry::Transaction::Operation
+      include DryTwitter::Import["repositories.users"]
 
       def call(input)
-        Success(input)
+        begin
+          users.create(user_name: input["user"]["user_name"], password: input["user"]["password"])
+          Success(input)
+        rescue Exception
```

Comment:
Global error handlers are bad, no matter the language, can you use dry-monads instead?